ui/src/app/settings/containers/Users/Users.js

```diff
@@ -14,54 +15,85 @@ import Pagination from "app/base/components/Pagination";
 
-const generateUserRows = (users, authUser) =>
-  users.map(user => ({
-    columns: [
-      {
-        content: <Link to={`/users/${user.id}`}>{user.username}</Link>,
-        role: "rowheader"
-      },
-      { content: user.email },
-      { content: 37, className: "u-align--right" },
-      { content: "Local" },
-      { content: "12 mins ago" },
-      { content: user.is_superuser ? "Admin" : null },
-      { content: user.sshkeys_count, className: "u-align--right" },
-      {
-        content: (
-          <>
-            <Link
-              to={`/users/${user.id}/edit`}
-              className="p-button--base is-small"
-            >
-              <i className="p-icon--edit">Edit</i>
-            </Link>
-            {user.id !== authUser.id && (
-              <Link
-                to={`/users/${user.id}/delete`}
-                className="p-button--base is-small"
+const generateUserRows = (users, authUser, expandedId, setExpandedId) =>
+  users.map(user => {
+    const expanded = expandedId === user.id;
+    return {
+      className: expanded ? "user-list__expanded-row" : null,
+      columns: [
+        {
+          content: <Link to={`/users/${user.id}`}>{user.username}</Link>,
+          role: "rowheader"
+        },
+        { content: user.email },
+        { content: 37, className: "u-align--right user-list__cell--small" },
+        { content: "Local", className: "user-list__cell--small" },
+        { content: "12 mins ago" },
+        {
+          content: user.is_superuser ? "Admin" : null,
+          className: "user-list__cell--small"
+        },
+        {
+          content: user.sshkeys_count,
+          className: "u-align--right user-list__cell--small"
+        },
+        {
+          content: (
+            <>
+              <Button
+                appearance="base"
+                element={Link}
+                to={`/users/${user.id}/edit`}
+                className="is-small user-list__button"
               >
-                <i className="p-icon--delete">Delete</i>
-              </Link>
-            )}
-          </>
-        ),
-        className: "u-align--right"
+                <i className="p-icon--edit">Edit</i>
+              </Button>
+              {user.id !== authUser.id && (
+                <Button
+                  appearance="base"
+                  className="is-small user-list__button"
+                  onClick={() => setExpandedId(user.id)}
+                >
+                  <i className="p-icon--delete">Delete</i>
+                </Button>
+              )}
+            </>
+          ),
+          className: "u-align--right user-list__cell--small"
+        }
+      ],
+      expanded: expanded,
+      expandedContent: expanded && (
+        <div className="user-list__expanded-content">
+          <Row>
+            <Col size="8">
+              Are you sure you want to delete user "{user.username}"?{" "}
+              <span className="u-text--light">
```

Comment:
This seems like the main message. I don't think we should be making it subtle.

Reply:
This matches the design here: https://app.zeplin.io/project/5ce5423a25863c1e0d7abb27/screen/5d36e2233615fbb2789ce87f.

Sorry just realised I didn't link to the design in the PR.